Arista/Ticks/disconnects.tick

```diff
@@ -1,7 +1,7 @@
 var UPDATES_IN_DISCONNECT_CHECK = 240
 var UPDATES_IN_ONE_DISCONNECT = 40
 var DISCONNECTS_CHECK_PERIOD = 1h
-var HOST_CHECK_PERIOD = 1m
+var HOST_CHECK_PERIOD = 15m
```

Comment:
Can these constants be defined in a common file and import them in the tick files? All the ticks seem to redefine the same constants.